style.css

```diff
@@ -0,0 +1,386 @@
+.menu {
+  display: flex;
+  justify-content: right;
+  padding: 20px;
+}
+
+.page {
+  display: flex;
+  flex-direction: column;
+  margin: unset;
+  overflow: auto;
+  margin-right: 24px;
+  margin-left: -24px;
+}
+
+.greetings {
+  justify-content: center;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+}
+
+.job {
+  color: #36b37e;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+}
+
+.socials {
+  display: flex;
+  gap: 1rem;
+  justify-content: center;
+  list-style-type: none;
+  margin-top: 14px;
+  padding-bottom: 2px;
+}
+
+.button {
+  color: #172b4d;
+  justify-content: center;
+  display: flex;
+  padding-bottom: 10px;
+}
+
+.headline {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 46px;
+  text-align: center;
+  overflow-y: auto;
+}
+
+.section-container9 {
+  background-image: url("images/illustrator-mobile.svg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+}
+
+.background-image {
+  width: 100%;
+  position: absolute;
+}
+
+.line-break::after {
+  content: '';
+  width: 48px;
+  height: 4px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 24px;
+  background: #36b37e;
+  border-radius: 24px;
+  display: flex;
+  justify-content: center;
+}
+
+.works {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  padding-top: 40px;
+}
+
+.section-placeholder {
+  margin-top: 50px;
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 58px;
+  margin-right: 24px;
+  margin-left: 24px;
+  align-items: center;
+  padding: 64px 24px 64px 24px;
+}
+
+.placeholder-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 10px;
+}
+
+.text-container {
+  border: 1px solid #ebf0ee;
+  width: 100%;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 12px;
+  margin-top: -10px;
+  background: white;
+}
```

Comment:
- [ ] Kindly make sure to use the appropriate font given in the Figma design as I noticed some of the texts in the works section are using the browser's default font. ✔
- Your design:
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/104968236/aecce614-45bd-45e4-845d-8374a8fb30f2)
- Figma design:
  ![image](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/104968236/e889b594-e718-436d-b2fb-64b360a030a0)
- The above screenshot only highlights the project card but you should make sure all the other texts are also using the proper font including the ones in the "About me" section.